db/sqlinterfaces.c

```diff
@@ -7677,24 +7665,24 @@ static int process_set_commands(struct sqlclntstate *clnt)
     num_commands = sql_query->n_set_flags;
     for (int ii = 0; ii < num_commands && rc == 0; ii++) {
         sqlstr = sql_query->set_flags[ii];
-        sqlstr = cdb2_skipws(sqlstr);
```

Comment:
I think it'd be better to offload this whole parsing of SET command to the parser, so that we will no longer have to worry about spaces, cases, etc. Not something related to this PR obviously, just a note.

Reply:
Could not agree more.